tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -41,90 +41,90 @@ test_i2c_iaqcore: iaqcore@5 {
 	reg = <0x5>;
 };
 
-test_i2c_bme280: bme280@7 {
+test_i2c_bme280: bme280@6 {
```

Comment:
Nit - add a comment to the file header indicating that the I2C peripheral addresses should be sequential.

Reply:
Done